index.js

```diff
@@ -0,0 +1,31 @@
+import navSection from './modules/navsection.js';
+import displayBook from './modules/displaybooks.js';
+import allBooks from './modules/awesomebooks.js';
+import { DateTime } from './modules/luxon.js';
+
+import { addToLocalStorage, getFromLocalStorage } from './modules/localstorage.js';
+
+document.addEventListener('DOMContentLoaded', () => {
+  navSection();
+  displayBook();
+});
+
+// add-book
+const addBtn = document.getElementById('addButton');
+addBtn.addEventListener('click', () => {
+  const title = document.getElementById('title');
+  const author = document.getElementById('author');
+  allBooks.bookObj(title.value, author.value);
+  addToLocalStorage();
+});
+
```

Comment:
- [ ] I think it would be better if you implement a feature which prevent a user to submit an empty form. Right now, even if you add the `required` attribute from html file, an empty book is always added once clicked the add button